CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -191,16 +193,19 @@ public let DECL_NODES: [Node] = [
       Child(
         name: "modifiers",
         kind: .collection(kind: .declModifierList, collectionElementName: "Modifier", defaultsToEmpty: true),
-        nameForDiagnostics: "modifiers"
+        nameForDiagnostics: "modifiers",
+        documentation: "Modifiers that are attached to the actor declaration."
```

Comment:
```suggestion
        documentation: "Modifiers like `public` that are attached to the actor declaration."
```

Same for the other documentation of modifiers.